pkg/controller/compliancescan/scan.go

```diff
@@ -525,9 +527,24 @@ func (r *ReconcileComplianceScan) reconcileReplicatedTailoringConfigMap(scan *co
 	origCM := &corev1.ConfigMap{}
 	origKey := types.NamespacedName{Name: origName, Namespace: origNs}
 	err := r.client.Get(context.TODO(), origKey, origCM)
+	// Tailoring ConfigMap not found
 	if err != nil && errors.IsNotFound(err) {
+		// We previously had dealt with this issue, just requeue
+		if strings.HasSuffix(scan.Status.ErrorMessage, tailoringNotFoundPrefix) {
```

Comment:
Are you sure this should be a `HasSuffix` not a `HasPrefix` ?

Reply:
you're right. Thanks for catching this!